Sources/Features/DappsAndPersonas/DappDetails/DappDetails.swift

```diff
@@ -222,7 +222,7 @@ public struct DappDetails: Sendable, FeatureReducer {
 			}
 
 			// FIXME: Can a dApp be associated with XRD
```

Comment:
Regarding old comment: `// FIXME: Can a dApp be associated with XRD` - product decision by Matt I guess? But IMO it would be nice if XRD is associated with the Dashboard dapp? :)